### PR TITLE
Publish "next" Helm chart documentation for tempo-distributed

### PR DESCRIPTION
**Pull request by jdbaldry**
See individual commits.

### CODE REVIEW

File: docs/sources/helm-charts/tempo-distributed/release-notes/_index.md [hunk 1]
```diff
@@ -3,12 +3,11 @@ description: Release notes for tempo-distributed Helm chart
...
 keywords:
   - Helm chart
   - Grafana Tempo
 ---
 
 # Grafana Tempo Helm chart release notes
 
-{{< section menuTitle="true" >}}
\ No newline at end of file
+{{< section menuTitle="true" >}}
```
Comment (knylander-grafana) [inline docs/sources/helm-charts/tempo-distributed/release-notes/_index.md:13]:
```suggestion
<!-- 
Helm chart release notes currently live in the helm-charts repo. 
Use this short code if/wnen the release notes are moved to live with the doc. 
{{< section menuTitle="true" >}} 
-->

The release notes for the tempo-distributed Helm chart currently reside in the [tempo-distributed README file](https://github.com/grafana/helm-charts/blob/main/charts/tempo-distributed/README.md).

```

**Review by knylander-grafana — COMMENTED** [review]
Thank you for removing draft:true. I updated the release notes page with a link to the actual release notes for the tempo-distributed Helm chart.

**Review by knylander-grafana — APPROVED** [review]
LGTM

File: docs/sources/helm-charts/tempo-distributed/release-notes/_index.md [hunk 2]
```diff
@@ -3,12 +3,18 @@ description: Release notes for tempo-distributed Helm chart
...
   - Grafana Tempo
 ---
 
 # Grafana Tempo Helm chart release notes
 
-{{< section menuTitle="true" >}}
\ No newline at end of file
+<!-- 
+Helm chart release notes currently live in the helm-charts repo. 
+Use this short code if or when the release notes are moved to live with the doc.
```
Comment (knylander-grafana) [inline docs/sources/helm-charts/tempo-distributed/release-notes/_index.md:15]:
```suggestion
Use this short code if or when the release notes are moved to live with the doc in the tempo repo. 
```

**Review by osg-grafana — APPROVED** [review]

**Review by zalegrala — APPROVED** [review]
Looks good to me, thanks for putting this together.

**Pull request merged**